packages/legend-graph/src/MetaModelConst.ts

```diff
@@ -320,3 +320,16 @@ export enum MILESTONING_STEROTYPES {
 }
 
 export const LATEST_DATE = '%latest';
+
+export enum DEFAULT_MILESTONING_PARAMETERS {
```

Comment:
Let's split these into `DEFAULT_PROCESSING_DATE_MILESTONING_PARAMETER_NAME` and `DEFAULT_BUSINESS_DATE_MILESTONING_PARAMETER_NAME`

Reply:
Also, please do me a favor and rename `MILESTONING_STEROTYPES` to `MILESTONING_STEROTYPE`